packages/clerk-js/src/ui/elements/ApplicationLogo.tsx

```diff
@@ -66,7 +66,18 @@ export const ApplicationLogo = (props: ApplicationLogoProps) => {
         props.sx,
       ]}
     >
-      {logoUrl ? <RouterLink to={logoUrl}>{image}</RouterLink> : image}
+      {logoUrl ? (
+        <RouterLink
+          sx={{
+            justifyContent: 'center',
+          }}
+          to={logoUrl}
+        >
+          {image}
+        </RouterLink>
+      ) : (
+        image
+      )}
```

Comment:
@octoper is this centering only if there is a logoUrl? What happens when we only have an image?

Reply:
This already works as we had `justifyContent: 'center'` in the outer element https://github.com/clerk/javascript/pull/2591/files#diff-4a9926d4df94a1d58e1ff0f95a12fa0399f27a30212873d2bc3d96a8842abb0eL57